eduNEXT/eox-audit-model
# perf(DS-368): add support for olive Open edX release

## Description

This PR adds support for the olive release maintaining the current support for nutmeg. It was tested on the two local environments.

## Testing instructions

Setting up the environment:
1. Install eox-core in LMS/LMS worker:
```
pip install git+https://github.com/eduNEXT/eox-core.git@MJG/DS-364-olive-compatibility#egg=eox_core==8.0.0_1
```
2. Install eox-audit-model: 
```
pip install git+https://github.com/eduNEXT/eox-audit-model.git@MJG/olive-support#egg=eox_audit_model==2.0.0_1
```
3. Migrate eox-audit-model: `python manage.py lms migrate eox_audit_model` (LMS)
4. Restart services
5. Add `CELERY_ALWAYS_EAGER = False` to your development settings.

Generating audit entries using eox-core:
1. Go to http://lms-url/eox-core/api-docs
2. Create a user and then create the same user for the 2nd time
3. Go to /admin and search for `audit`. You'll see the new record generated by the transaction.

## Additional information

Detailed test cases:
https://docs.google.com/document/d/1v6mxFSf1k7JrxU36Ok9uVDFBhtxVuGmhFLtxtbIsPFs

## Checklist for Merge
